src/renderers/native/__tests__/ReactNativeMount-test.js

```diff
@@ -108,4 +108,20 @@ describe('ReactNative', () => {
     ReactNative.render(<Component chars={after} />, 11);
     expect(UIManager.__dumpHierarchyForJestTestsOnly()).toMatchSnapshot();
   });
+
+  it('calls setState with the correct parameters', () => {
```

Comment:
Let's add a similar test to `ReactTestUtils-test` for shallow renderer, and to `ReactCompositeComponent-test` for DOM renderer.